apps/core-create-react-app/package.json

```diff
@@ -1,25 +1,28 @@
 {
-  "name": "my-app",
+  "name": "core-create-react-app",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
     "@clr/city": "^1.1.0",
-    "@clr/core": "^3.1.0",
+    "@clr/core": "./packages/core/dist/core",
+    "@clr/react": "./packages/react/dist/react",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.4.1",
     "@testing-library/user-event": "^7.2.1",
     "@types/jest": "^24.9.1",
     "@types/node": "^12.12.29",
     "@types/react": "^16.9.23",
     "@types/react-dom": "^16.9.5",
+    "jest": "24.9.0",
+    "normalize.css": "^8.0.1",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
     "react-scripts": "3.4.1",
     "typescript": "^3.7.5"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
+    "build": "SKIP_PREFLIGHT_CHECK=true react-scripts build",
```

Comment:
Create react app does not like how yarn resolves jest. This skips is validation check. I figured that would be ok for this demo since we are not adding tests here.